src/Styleguide/Artwork/Banner.tsx

```diff
@@ -1,22 +1,62 @@
 import React from "react"
 import styled from "styled-components"
+import {
+  flexDirection,
+  alignItems,
+  alignContent,
+  justifyContent,
+  space,
+} from "styled-system"
 import { Avatar } from "../Elements/Avatar"
+import { Sans, Serif } from "../Elements/Typography"
 
-const Flex = styled.div`
+interface FlexProps {
+  flexDirection?: "row" | "column"
+  alignItems?: "flex-start" | "flex-end" | "center" | "baseline" | "stretch"
+  alignContent?:
+    | "flex-start"
+    | "flex-end"
+    | "center"
+    | "space-between"
+    | "space-around"
+    | "stretch"
+  justifyContent?:
+    | "flex-start"
+    | "flex-end"
+    | "center"
+    | "space-between"
+    | "space-around"
+    | "space-evenly"
+  m?: string
+  mr?: string
+  ml?: string
+  mt?: string
+  mb?: string
+}
+
+const Flex = styled.div.attrs<FlexProps>({})`
   display: flex;
-  flex-direction: row;
+  ${flexDirection};
+  ${alignItems};
+  ${alignContent};
+  ${justifyContent};
+  ${space};
 `
 
 export class Banner extends React.Component {
   render() {
     return (
-      <Flex>
+      <Flex flexDirection="row">
         <Avatar size="110px" src="https://picsum.photos/110/110/?random" />
-        <div>
-          <div>In show</div>
-          <div>Francesca DiMattio: Boucherouite</div>
-          <div>Salon 94</div>
-        </div>
+        <Flex flexDirection="column" justifyContent="center" ml="20px">
+          <Sans weight="medium" size="2">
+            In show
+          </Sans>
+          <Serif size="4t">Francesca DiMattio: Boucherouite</Serif>
+          <Serif size="4t" color="black60">
+            Salon 94
+          </Serif>
+        </Flex>
```

Comment:
Ah, now I understand why this `<Flex />` is broken out -- so that composing elements can remain neutral and live within the system. @yuki24 did something similar over in reaction. 

Would you mind pushing this into a separate component before merge?

Reply:
Sure thing. Elements?

Reply:
👍